### DISCUSSION TITLE
Would Like to set up RXN Model for Reforming

### DISCUSSION THREAD

**Discussion by feeburn** [ReactionMechanismGenerator/ReactionMechanismSimulator.jl#179] [Q&A]
Dear RMSers,
I have a list of ~200 feed species and their mol/s rate.  Could I use RMG/RMS to create the rxn net work and then solve the LFER parameters based on species in the reactor effluent (mol/s)?  Also is there an energy balance option so that I can match the temperature of the effluent leaving reactor?

**Comment by mjohnson541** [comment]
Sorry this is very late, I wasn't even aware this discussion section existed before now. 

I'm not familiar with LFER parameters, but I believe this is sensible. The "energy balance" option in Chemkin for a batch or PFR reactor is analogous in RMS gas phase to choosing a constant volume domain (which will fix volume and solve the energy equation to determine pressure and temperature as a function of time). You can also use a ParameterizedTPReactor to set the pressure and temperature to specific profiles during the reaction.
